src/index.html

```diff
@@ -1,66 +1,103 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset='UTF-8'>
     <title>Activity Tracker</title>
-    <link href="https://fonts.googleapis.com/css?family=Open%20Sans" rel='stylesheet'>
-    <link rel="stylesheet" type="text/css" href="./styles.css">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Open%20Sans' rel='stylesheet'>
+    <link rel='stylesheet' type='text/css' href='./styles.css'>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
   </head>
-  <body class="body">
+  <body class='body'>
     <header class= 'header'>
-     <div class='greeting'>Welcome, Name!</div>
-     <div class='date'>Date</div>
+     <h2 class='greeting'>Welcome, Name!</h2>
+     <h2 class='date'>Date:</h2>
     </header>
     <section class='widgets'>
       <div class='left-column columns'>
         <div class='user-card-widget'>
           <h1 class='user-card-title title'>User Card</h1>
-          <h2 class="user-card-id"></h2>
-          <h2 class="user-card-name"></h2>
-          <h2 class="user-card-address"></h2>
-          <h2 class="user-card-email"></h2>
-          <h2 class="user-card-stride-length"></h2>
-          <h2 class="user-card-daily-step-goal"></h2>
-          <h2 class="user-card-friends"></h2>
+          <div class='uc-module-id uc-module'>
```

Comment:
Each module within the widgets is now within its own div. Specific classes were added within these divs for accessibility. This allows for more efficient formatting. 

No class names that we accessed in functions were changed, but we should double check all querySelectors before further DOM manipulation.

Reply:
This is good stuff!